src/main/scala/com/imaginea/activegrid/core/models/ImageInfo.scala

```diff
@@ -8,28 +9,30 @@ import org.slf4j.LoggerFactory
   * Created by sampathr on 22/9/16.
   */
 case class ImageInfo(override val id: Option[Long],
```

Comment:
id is optional in all case classes as we assign the value to it  after getting the id by saving it to graph db